packages/app/src/__deprecated__/renderer/utils/form-validators.ts

```diff
@@ -96,6 +96,14 @@ export const emailValidator: RegisterOptions = {
   required: intl.formatMessage({ id: "component.formErrorRequiredEmail" }),
 }
 
+export const emailValidatorNotRequired: RegisterOptions = {
+  pattern: {
+    value:
+      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
```

Comment:
please moved regex to const variable instead  of copy value 😉

Reply:
Moved in [d45f7ad](https://github.com/mudita/mudita-center/pull/879/commits/d45f7ad849a3c13f6ed852b320bc75176bb25510)